peerconnection.go

```diff
@@ -1160,16 +1167,23 @@ func (pc *PeerConnection) startRTPReceivers(incomingTracks []trackDetails, curre
 func (pc *PeerConnection) startRTPSenders(currentTransceivers []*RTPTransceiver) {
 	for _, transceiver := range currentTransceivers {
 		// TODO(sgotti) when in future we'll avoid replacing a transceiver sender just check the transceiver negotiation status
-		if transceiver.Sender() != nil && transceiver.Sender().isNegotiated() && !transceiver.Sender().hasSent() {
```

Comment:
again, `Sender()` can be mutated during execution here. the fix i made is messy. maybe there is a better way?